assets/js/components/Dashboard/NewPaMessage/StationsAndZones/SelectZonesPage.tsx

```diff
@@ -0,0 +1,541 @@
+import React, {
+  Fragment,
+  ReactNode,
+  useEffect,
+  useMemo,
+  useState,
+} from "react";
+import { Page } from "../types";
+import { Button } from "react-bootstrap";
+import { Place } from "Models/place";
+import fp from "lodash/fp";
+import { Screen } from "Models/screen";
+import { getZoneLabel, signIDs, sortByStationOrder } from "../../../../util";
+import cx from "classnames";
+import { ArrowLeftShort, ArrowRightShort, Dot } from "react-bootstrap-icons";
+import { useRouteToRouteIDsMap } from "../hooks";
+import { Dictionary } from "lodash";
```

Comment:
**suggestion:** is this just a type import? can we make it a type import then?

```suggestion
import type { Dictionary } from "lodash";
```

or

```suggestion
import { type Dictionary } from "lodash";
```

should both work